packages/@sanity/portable-text-editor/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "@sanity/portable-text-editor",
+  "version": "2.14.0",
+  "description": "Portable Text Editor made in React",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "author": "Sanity.io <hello@sanity.io>",
+  "engines": {
+    "node": ">=8.0.0"
```

Comment:
is this accurate? I see now that the rest of the packages in here has >= 10.0.0 which is out of LTS, so we should probably change this across packages.